src/pages/Home.jsx

```diff
@@ -125,37 +144,42 @@ const Home = (props) => {
-                    {' '}
-                    For help determining your data&apos;s permitted uses, try our
-                    {' '}
-                    <Link to="/consent_text_generator">Data Sharing Language Tool</Link>, which follows GA4GH guidelines.
-                  </p>
-                ]}
-                moreContent={[
-                  <div key="p-1">
-                    <p style={paragraph}>As part of our efforts to enhance collaborative research, the Broad Institute developed the “Data Use Oversight System” (DUOS) to semi-automate and efficiently manage compliant sharing of human subjects data. DUOS&apos; objective is two-fold, to enhance data access committee&apos;s confidence that data use restrictions are respected while efficiently enabling appropriate data access.</p>
-                    <p style={paragraph}>To better enable the use of existing human subjects datasets in future projects, DUOS mimics, in a semi-automated fashion, the data access request review processes common to DACs globally, like those in dbGaP. To this end, the system includes interfaces to capture and structure data use restrictions and data access requests as machine-readable data use terms based on the GA4GH&apos;s Data Use Ontology. With these machine-readable terms for dataset&apos;s use limitations and data access requests established, DUOS is able to trigger a matching algorithm to reason if data access should be granted given the research purpose and the data restrictions, serving as a decision support tool for DACs using DUOS.</p>
-                    <p style={paragraph}>To evaluate the feasibility of using machine readable data use terms to interpret data use restrictions and access requests, we are piloting a trial of DUOS overseen by Partners’ Healthcare IRB. During the pilot, DACs comprised of governmental and non-governmental data custodians will pilot the use of DUOS, its ability to structure use limitations and access requests, and the accuracy of the DUOS algorithm. This aids us in improving the DUOS algorithm and providing feedback on the GA4GH Data Use Ontology based on experts’ feedback.</p>
+
+        <section style={{ margin: '5rem auto', padding: '0 2rem' }}>
+          <div style={{ maxWidth: '1200px', margin: '0 auto', textAlign: 'center' }}>
+            <p style={header}>Data Libraries in DUOS</p>
+            <p style={description}>Click the images below to view curated Data Libraries, and search and request access to data.</p>
+
+            <div style={logoGrid} className="logo-grid">
+              <OverflowTooltip id="anvil" tooltipText="AnVIL">
+                <Link to="/datalibrary/anvil" style={{ textDecoration: 'none' }}>
```

Comment:
The link behavior is confusing if the user is not signed in. For example, if you are not signed in, you are redirected back to this page, but not prompted in any way so it looks like nothing happened. After signing in, the user is directed to the correct link. I think we need some kind of prompt on these images if the user is not signed in.

Reply:
Yeah I agree. I'll fix this.

Reply:
@rushtong @solideoglori I’ve added a visual indicator on the cards to show that these data libraries require login to access. However, after logging in the UI always redirects users to the admin_dar_collections_page, which prevents them from scrolling the dataset they want to view. This feature may need some more thinking or I am missing something.

I added some visual indicators for unauthenticated users and disabled navigation. 

![Screenshot From 2025-05-20 11-52-23](https://github.com/user-attachments/assets/fdff2d12-17e9-4951-be15-b206f785b1ee)